lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -3293,6 +3293,11 @@ void __cdecl _alloca_probe_16()
 
         Assert(!(callInfo.Flags & CallFlags_New));
 
+        if (!JavascriptOperators::IsObject(args[0]))
```

Comment:
yup - we should move fromVar after isCallable check.

---
In reply to: [126812211](https://github.com/Microsoft/ChakraCore/pull/3320#discussion_r126812211) [](ancestors = 126812211)